customer-terminate-service/app/src/main/java/se/inera/intyg/cts/infrastructure/service/SendPasswordImpl.java

```diff
@@ -0,0 +1,42 @@
+package se.inera.intyg.cts.infrastructure.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import se.inera.intyg.cts.domain.model.Termination;
+import se.inera.intyg.cts.domain.repository.TerminationRepository;
+import se.inera.intyg.cts.domain.service.SendPassword;
+import se.inera.intyg.cts.infrastructure.integration.SendSMS;
+import se.inera.intyg.cts.infrastructure.integration.tellustalk.dto.SMSResponseDTO;
+
+@Service
+public class SendPasswordImpl implements SendPassword {
+
+    private static final Logger LOG = LoggerFactory.getLogger(SendPasswordImpl.class);
+
+    private final SendSMS sendSMS;
+
+    private final String smsActive;
+
+    private final TerminationRepository terminationRepository;
+
+    public SendPasswordImpl(SendSMS sendSMS, TerminationRepository terminationRepository, @Value("${sms.active}") String smsActive) {
+        this.sendSMS = sendSMS;
+        this.terminationRepository = terminationRepository;
+        this.smsActive = smsActive;
+    }
+
+    @Override
+    public void sendPassword(Termination termination) {
+        if (Boolean.TRUE.toString().equals(smsActive)) {
```

Comment:
Move smsActive to application service MessageService. Rename to send.password.active. 

If the feature isn't active, it should not make any changes to the Termination.